mastercraft.cmd

```diff
@@ -113,7 +114,21 @@ include mc_include.cmd
      var NOWO $MC_%society.type_NOWO
      gosub clearvolume
 
+     ### If main.storage is added to tied tools, assume it is tied.
+     if matchre("%tiedtools", %main.storage) then
+     {
+          put tap bag
+          match untiebag toolbelt
+          match donebag wearing
+          matchwait
+
+          untiebag:
+          gosub PUT untie %main.storage 
+          gosub PUT wear %main.storage
 
+          donebag:
+          
+     }

```

Comment:
Okay this one.
I added support for a bag tied to a toolbelt. IF you add the verb to your tied tools that is also your main.storage, it will untie the verb and wear it before your start mastercraft. After it is done mastercraft it will remove it and tie it.